scripts/compose/.env-default

```diff
@@ -0,0 +1,12 @@
+
+# Default values
+HOPRD_DATA=/app/hoprd-db
+HOPRD_INIT=true
+HOPRD_API=true
+HOPRD_ANNOUNCE=true
+HOPRD_API_HOST=0.0.0.0
+HOPRD_IDENTITY=/app/hoprd-db/.hoprd.id
+HOPRD_HEALTH_CHECK=true
+HOPRD_HEALTH_CHECK_HOST=0.0.0.0
+HOPRD_HEARTBEAT_INTERVAL=20000
+HOPRD_HEARTBEAT_THRESHOLD=60000
```

Comment:
10 minutes? In Providence the max timeout is like 30s. These environment variables and other stuff will need to be revisited and removed, they are no longer reflected in the libp2p protocols.

Reply:
Got it, will take into consideration for next patches or release